packages/react-pose/src/components/Transition/index.tsx

```diff
@@ -19,17 +20,39 @@ class Transition extends React.Component<Props, State> {
     props: Props,
     { isLeaving, removeFromTree, children }: State
   ) => {
-    const incomingChildren = makeChildList(props.children);
+    const {
+      children: groupChildren,
+      preEnterPose,
+      enterPose,
+      exitPose,
+      flipMove,
+      animateOnMount,
+      ...propsForChild,
+    } = props;
 
-    return {
+    const incomingChildren = makeChildList(groupChildren);
+
+    if (process.env.NODE_ENV !== 'production') {
```

Comment:
Will this be correctly transpiled/minified away in prod?

Reply:
yeah